.changelog/unreleased/breaking-changes/1146-mempool-remove-ids.md

```diff
@@ -0,0 +1,3 @@
+- `[mempool]` Remove `mempoolIDs` for internally storing peer ids as `p2p.ID`
+  instead of `uint16`.
+  ([\#1146](https://github.com/cometbft/cometbft/pull/1146))
```

Comment:
I think it's mostly important just to mention which public Go APIs (`mempool.Reactor.InitPeer` and `mempool.Reactor.RemovePeer`) have been removed.

Reply:
They haven't been removed. They were redundant because now they are both empty, which is equal to the default behaviour provided by `BaseReactor`.